.github/workflows/pr.yaml

```diff
@@ -73,6 +73,15 @@ jobs:
       - name: unit test
         run: make test
 
+      - name: setup python
```

Comment:
Why does the conftest repo need to install this in the PR flow?

Reply:
Thanks. I am OK with this here since the PR workflow doesn't have permissions to do anything in the conftest repo.